README.md

```diff
@@ -1801,7 +1801,9 @@ arXiv, 2019. [PDF](https://arxiv.org/pdf/1904.04805), [Video](https://neurorobot
 - <a name="Gehrig20cvpr"></a>Gehrig, D., Gehrig, M., Hidalgo-Carrió, J., Scaramuzza, D.,  
 *[Video to Events: Recycling Video Datasets for Event Cameras](http://rpg.ifi.uzh.ch/docs/CVPR20_Gehrig.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020. [YouTube](https://youtu.be/uX6XknBGg0w),  [Project page](https://github.com/uzh-rpg/rpg_vid2e)
-
+- <a name="Deng21eventkd"></a>Y. Deng, H. Chen, H. Chen and Y. Li,  
```

Comment:
Hello.
Could you please follow the [convention](https://github.com/uzh-rpg/event-based_vision_resources/blob/master/Contributing.md)?
key: Deng21eventkd -> Deng21tip (unless it is already taken)
Names: Y. Deng, H. Chen, ... -> the other way around: Deng, Y., Cheng, H., ...
IEEE Transactions on Image Processing -> IEEE Trans. Image Process.  (IEEE abbreviation).

Reply:
Of course, thanks for your notice